TLM/.editorconfig

```diff
@@ -1,9 +1,77 @@
-﻿root = true
-
-[*.{cs,vb}]
-indent_style = space
-indent_size = 4
-trim_trailing_whitespace = true
+root=true
 
 [*.cs]
-# csharp_new_line_before_open_brace = methods
+max_line_length=100
+tab_width=8
+
+[*]
+charset=utf-8
+end_of_line=crlf
+trim_trailing_whitespace=true
+insert_final_newline=true
```

Comment:
Would it be better to keep `insert_final_newline` just for `.cs` files? I'm not sure what effect it will have on some of the `.txt` files, eg. it would probably break the mod checker if someone adds blank line to `incompatible_mods.txt`.

Reply:
I case of empty lines in `incompatible_mods` would be nice to add little fix for that

Reply:
Moved up to .cs

Reply:
@krzychu124 I'm about to update the mod checker anyway so I'll include that fix.